IRremoteESP8266.cpp

```diff
@@ -470,7 +470,7 @@ void ICACHE_FLASH_ATTR IRsend::sendRCMM(uint32_t data, uint8_t nbits) {
 void ICACHE_FLASH_ATTR IRsend::sendPanasonic64(unsigned long long data,
                                                unsigned int nbits,
                                                unsigned int repeat) {
-  enableIROut(37);  // Set IR carrier frequency
+  enableIROut(36700U);  // Set IR carrier frequency of 36.7kHz.
   IRtimer usecTimer = IRtimer();
 
   for (uint16_t i = 0; i <= repeat; i++) {
```

Comment:
Phew. It was 3am when I wrote it, and about the same time, when I read/wrote the feedback. I was going to chalk it up to being to tired to see straight.

Anyway, always glad to double check etc. That's one of the points of a review.